index.js

```diff
@@ -22,7 +22,7 @@ import Sheet from './components/Sheet';
 import Button from './components/Button';
 
 const ANDROID_KIT_KAT_SDK_VERSION = 19;
-const androidPermissionRequestRequired = Platform.Version < ANDROID_KIT_KAT_SDK_VERSION;
+const androidPermissionRequestRequired = parseInt(Platform.Version) < ANDROID_KIT_KAT_SDK_VERSION;
```

Comment:
This is causing the lint to crash in the CI 🤔 

```suggestion
const androidPermissionRequestRequired = parseInt(Platform.Version, 10) < ANDROID_KIT_KAT_SDK_VERSION;
```

Reply:
Updated the PR as suggested.